.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "python3_ws/src/calibration/scripts/resources/depthai-boards"]
-	path = python3_ws/src/calibration/scripts/resources/depthai-boards
+	path = python3_ws/src/calibration/scripts/resources/depthai_boards
 	url = https://github.com/luxonis/depthai-boards
+[submodule "python3_ws/src/calibration/scripts/resources/depthai-boards-private"]
+	path = python3_ws/src/calibration/scripts/resources/depthai_boards_private
+	url = https://github.com/luxonis/depthai-boards-private.git
```

Comment:
You may also do it by relative path, which will auto retain permissions/auth of parent